cvat-ui/src/components/header/header.tsx

```diff
@@ -62,6 +67,34 @@ function HeaderContainer(props: Props): JSX.Element {
                 >
                     Tasks
                 </Button>
+                <Popover placement="bottom" title={content} content={text} trigger="click" overlayStyle={{width : "50vw"}}>
```

Comment:
I agree with you about it should be a switchable window, but this component doesn't look pretty.
I believe a Modal window would be great here.
You know, a tool name, an image, description, a version, the github link, etc.

Reply:
Hi, @bsekachev  
What image do you want to include here? 
As of now, I have the tool name, version number and description (in the same order). Also, I believe adding a github link would make it redundant as it already appears on the header. Kindly let me know about these. Thanks :)

Reply:
Hi, @LiSa20120 

To be honest, I don't know what image can we use here. Probably it's better to skip this moment for now. 

I have some ideas about content of the dialog window:
- The name of the tool
- Tool description
- Server version (can get via core.server.about()) 
- Core version (can get via core.client.version)
- Release notes ("what's new") (just a link to the section of the ```CHANGELOG.md``` file)
- License information (a link to ```LICENSE``` file in the repository)
- "Need help?" (a link to the gitter chat)
- Maybe link to [Forum on Intel Developer Zone](https://software.intel.com/en-us/forums/intel-distribution-of-openvino-toolkit)

@nmanovic, what do you think?

Reply:
![about](https://user-images.githubusercontent.com/38404726/71620882-39cc6d00-2bf2-11ea-8f69-9d1d02ec7139.png)
@bsekachev Have a look at it.

Reply:
@bsekachev How are you converting md files to html?

Reply:
@bsekachev How are you converting the md files to html? (this is to include CHANGELOG.md)
Should we install [react-markdown](https://www.npmjs.com/package/markdown-react-js)?